app/views/orders/receipt.html.haml

```diff
@@ -47,10 +47,13 @@
         - if @order.has_subsidies?
           %td.currency= order_detail.wrapped_subsidy
         %td.currency=order_detail.wrapped_total
+  %tfoot
     %tr
       - colspan = @order.has_subsidies? ? 5 : 4
-      %td{colspan: colspan}
+      %td.hide-on-mobile{colspan: colspan}
         %strong= t('.total')
-      %td.currency= number_to_currency(@order_details.sum { |d| d.total || 0 })
+      %td.currency
+        .responsive-header Total
```

Comment:
You are correct, I should be using the locale.

This is necessary because the footer fields don't pull in the JavaScript to get the responsive headers added, and I didn't think the footers were likely to be consistent enough to be programmatically added. So it's explicit (here and on the cart).

But I'll change it to the locale.